pyvista/plotting/plotter.py

```diff
@@ -6216,7 +6231,7 @@ def orbit():
                 else:
                     self.render()
                 sleep_time = step - (time.time() - tstart)
-                if sleep_time > 0 and not self.off_screen:
+                if sleep_time > 0 and (hasattr(self, "off_screen") and not self.off_screen):
```

Comment:
Given this needed change, this method should probably be in `Plotter`.  But let's not do that for a typing fix.

Reply:
To make sure I understood correctly, are we keeping this change here?

Reply:
Given that this functionality only sleeps when `not self.off_screen`, it seems reasonable to me that we would just ignore this when it doesn't have this attribute.

Although the code organization smells bad here, but we shouldn't fix that in a typing PR.

I would vote to keep your change, maybe add a comment if you feel it is needed.

Reply:
And potential simplification to
```suggestion
                if sleep_time > 0 and not getattr(self, "off_screen", True):
```
but the counter-intuitiveness of the `True` default paired with `None` might mean we're better off with the original plus a comment.

> Although the code organization smells bad here, but we shouldn't fix that in a typing PR.

Wholeheartedly agree.

Reply:
I have just added the following comment `# 'off_screen' attribute is specific to Plotter objects.`